### PR TITLE
Added {audio,video}BitsPerSecond read-only attributes.

### PR DESCRIPTION
**Pull request by yell0wd0g**
Also Removed some stale todo-properties entries and reflowed html in that area.

Re. https://github.com/w3c/mediacapture-record/issues/43

### CODE REVIEW

**Comment by yell0wd0g** [comment]
@foolip, @mreavy can you please take a look?thx

**Comment by yell0wd0g** [comment]
I thought about publishing the consolidated audio and video bit rates. The application knows what was written in the `options`, but due to clamping or calculations based on `bitsPerSecond`, the individual values could be different. Perhaps an app left the decision on the actual values to the UA and is interested in retrieving the configured ones. For all that, I don't think `bitsPerSecond` should be there. WDYT?

**Comment by yell0wd0g** [comment]
@mreavy, I took the chance of this pull request to homogeneise the wording you mentioned in https://github.com/w3c/mediacapture-record/issues/43.

File: MediaRecorder.html [hunk 1]
```diff
@@ -215,52 +221,25 @@
...
+      <th>Property Name</th>
+      <th>Values</th>
+      <th>Notes</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr id="def-property-MimeType">
+      <td>MimeType</td>
+      <td><code>DOMString</code></td>
+      <td> The MIME type that was be selected as the container for recording. This entry includes all the parameters to the base MimeType. The UA should be able to play back any of the MIME types it supports for recording. For example, it should be able to display a video recording in the HTML &lt;video&gt; tag. The default value for this property is platform-specific.</td>
```
Comment (foolip) [inline MediaRecorder.html]: This says "was be selected" and both "MIME type" and "MimeType".

Comment (foolip) [inline MediaRecorder.html]: I see this was just moved and reformatted in this commit, but might want to fix anyway :)

**Comment by foolip** [comment]
I don't have anything to object here, as long as `audioBitsPerSecond` and `videoBitsPerSecond` are only added to the options dictionary, that's just matching Gecko and not opening up any new issues I think. Aggregate bitrates or exposing the actually chosen target bitrate might be trickier.

**Comment by mreavy** [comment]
@miguelao The wording looks good, thanks.

File: MediaRecorder.html (hunk 1 above)
Comment (yell0wd0g) [inline MediaRecorder.html]: done both, thanks!

**Comment by yell0wd0g** [comment]
Sorry I re-read what the original patch was stating:

>  If the user has not specified any, the Platform implementation is free to choose this value and it <em title="must" class="rfc2119">must</em> read zero.</dd>

And I realised that my comments where not making much sense. I removed that line and substituted with 

>  or the calculated one if the user has specified <code>bitsPerSecond</code>.

And my comments apply to such writing. Apologies :P

**Comment by yell0wd0g** [comment]
ping @foolip can you check the latest patch?

**Comment by foolip** [comment]
Changes look good, thanks!

**Pull request merged**